JavaScript/JavaScriptSDK/ajax/ajax.ts

```diff
@@ -140,9 +140,9 @@ module Microsoft.ApplicationInsights {
         private sendHandler(xhr: XMLHttpRequestInstrumented, content) {
             xhr.ajaxData.requestSentTime = dateTime.Now();
 
-            // Add correlation headers only for requests within the same domain (ignore the port number)
+            // Add correlation headers only for requests within the same host and port number
```

Comment:
It is :) `host` (in opposite to `hostname`) contains the port number. It's also covered by the unit test.